server/sonar-main/src/main/java/org/sonar/application/cluster/HazelcastCluster.java

```diff
@@ -154,6 +190,7 @@ public void close() {
         // Removing listeners
         operationalProcesses.removeEntryListener(operationalProcessListenerUUID);
         hzInstance.getClientService().removeClientListener(clientListenerUUID);
+      hzInstance.getCluster().removeMembershipListener(nodeDisconnectedListenerUUID);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)